<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgotPasswordWindow</class>
 <widget class="QMainWindow" name="ForgotPasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">	background-color: rgb(31, 149, 239);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendRequestButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">            QPushButton {
                border: 2px solid #8f8f91;
                border-radius: 15px; /* Nửa chiều rộng hoặc chiều cao của nút */
                background-color: #dcdcdc;
                min-width: 30px; /* Đảm bảo nút là hình vuông */
                min-height: 30px; /* Đảm bảo nút là hình vuông */
            }
            QPushButton:hover {
                background-color: #c6c6c6;
            }
            QPushButton:pressed {
                background-color: #a9a9a9;
            }</string>
    </property>
    <property name="text">
     <string>Send Request</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Forgot Password </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QLineEdit {
                border: 2px solid gray;
                border-radius: 10px;
                padding: 0 8px;
                background: white;
                selection-background-color: darkgray;
            }

</string>
    </property>
    <property name="placeholderText">
     <string>Enter your email</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
